google/cloud/dataproc_spark_connect/session.py

```diff
@@ -480,11 +545,30 @@ def _get_exiting_active_session(
 
         def getOrCreate(self) -> "DataprocSparkSession":
```

Comment:
We probably should not terminate session if stop is called on it either, as it can be in use by other notebooks, etc.